textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/DoubleField.java

```diff
@@ -0,0 +1,47 @@
+package edu.uci.ics.textdb.common.field;
+
+import edu.uci.ics.textdb.api.common.IField;
+
+public class DoubleField implements IField{
+    
+    private Double value;

```

Comment:
I think Jianfeng was suggesting not to create unnecessary java objects.  If we can implement the same feature using the primitive types such as "double", we don't need to use a class.  Can we do it?